### ISSUE TITLE
Proposal: triage open proposals

### ISSUE THREAD

**Issue by lucasgonze** [magma/magma#15421]
# Problem

There are 55 open proposals. These open proposals are clogging up the work queue. Most are stale, some are completed and can be closed, a few are waiting on attention.

# Solution

Triage every open proposal.

1. If it is just waiting on acceptance, raise a vote in the TSC.
2. If it is abandoned or stale and no longer an option, close it.
3. If it is in progress but the proposer needs prodding, prod them.
4. If it needs a label, add it.
5. If it needs further study by another community member, document the work.
6. Identify outstanding PRs that need to be merged and are mergeable with a reasonable amount of work.
7. Once initial triage is done, prepare documentation for a technical meeting to review the substance of more complex items.

5, 6, 7 should be about labeling, not deep technical work. Avoid absorbing engineering time. That will be invested post-1.9. No more than one hour of engineering time should be invested.

# Non-goals

Implementation of proposals

# Bid

Micky Kumar and Lucas Gonze will collaborate. They are asking $1,500.

Acceptance criteria: no open proposals that aren't still in progress.

# Update June 13 2024

In discussion regarding acceptance in the TSC meeting on Monday June 10 the consensus was that the work done to date doesn't match with the original acceptance criteria "no open proposals that aren't still in progress." However, that acceptance criteria was rejected after the fact because the triage work is always going to be ongoing. For the purposes of getting the mess of stale proposals under control, consensus was that the work done to date was sufficient. Two different acceptance criteria were requested:

1. Document the change, which is what this note is for
2. Organize a meeting to review the substance of the proposals that were closed

**Comment by lucasgonze** [comment]
Call for vote: Lucas, seconded by Jordan

**Comment by jblakley** [comment]
I vote yes

**Comment by lucasgonze** [comment]
That is three in favor, none against, none abstaining. The ayes have it.

**Comment by lucasgonze** [comment]
Worksheet: 
https://docs.google.com/document/d/1HZHjGjptNahwRCgYbdynjhjcsw6tXA-p88eGATFT--8/edit

**Comment by lucasgonze** [comment]
Mickey created a proposal tracker:
https://github.com/orgs/magma/projects/26/views/1

This should remain a thing over the long term.

**Comment by lucasgonze** [comment]
We reviewed this Kanban in the TSC today. It was well received and had solid consensus behind it.

In the future we will continue to use this Kanban on a weekly basis.

**Comment by lucasgonze** [comment]
Ready for acceptance vote.

**Comment by jblakley** [comment]
@lucasgonze what specifically are we accepting? The completion of the triage? The completion of `Acceptance criteria: no open proposals that aren't still in progress.` The kanban structure? If it is the triage, is #7 done?

**Comment by lucasgonze** [comment]
> what specifically are we accepting

Let's talk in real-time in today's TSC.

**Comment by lucasgonze** [comment]
See "Update June 13 2024" above

**Comment by lucasgonze** [comment]
- Acceptance motion: Lucas
- Second: Som
- Vote +1: Yogesh

The motion is carried

**Comment by jordanvrtanoski** [comment]
+1 from my side as well

**Comment by lucasgonze** [comment]
Next: create invoice

**Comment by mickymkumar** [comment]
Invoice has been sent from my end.

**Comment by lucasgonze** [comment]
And mine. It's in AP's hands.

**Comment by lucasgonze** [comment]
AP has sent an approved PO.

**Comment by mickymkumar** [comment]
No new update from my side.

Can someone can share accounting contact information so I can reach out?

**Comment by lucasgonze** [comment]
> No new update from my side.
> 
> Can someone can share accounting contact information so I can reach out?

Just to verify, did you enclose the PO number?

**Comment by mickymkumar** [comment]
> > No new update from my side.
> > Can someone can share accounting contact information so I can reach out?
> 
> Just to verify, did you enclose the PO number?

No

**Comment by lucasgonze** [comment]
What you want to do is reissue the invoice referencing the PO number they emailed.

**Comment by mickymkumar** [comment]
> What you want to do is reissue the invoice referencing the PO number they emailed.

I can resend again the invoice but there is no PO Number created yet

**Comment by mickymkumar** [comment]
**The problem is in lf contract there used the wrong name in company and there are fixing it.**

**I CANNOT sign off until fix the with the correct name. -> It is not even a 5 Minute Job for them and it been a month saying the same thing**
